mmv1/templates/terraform/encoders/compute_backend_bucket.go.erb

```diff
@@ -0,0 +1,35 @@
+// This custom encoder helps prevent sending 0 for clientTtl, defaultTtl and
+// maxTtl in PATCH calls when unset in the provider (doing so results in an API
+// level error)
+c, cdnPolicyOk := d.GetOk("cdn_policy")
+
+// Only apply during update
+if !cdnPolicyOk {
+	return obj, nil
+}
+
+// Don't do anything if we're not PATCHing
+if obj["cdnPolicy"] == nil {
```

Comment:
is this how you were expecting to reference the object?

Reply:
This makes sense to me